### DISCUSSION TITLE
Error when using DAPimpleFoam

### DISCUSSION THREAD

**Discussion by erkhans** [mdolab/dafoam#591] [Bug]
Hello. I am getting this error, when I try to run the case with DAPimpleFoam. How I can set timeOpearator and solve this issue? Could you help me, please?

--> FOAM FATAL ERROR: 
calcUnsteadyObjFuncs is called but the timeOperator is not set!!! Options are: average or sum

    From function 
    in file DASolver/DASolver.C at line 187.

FOAM aborting

**Comment by friedenhe** [comment]
Please check: https://github.com/mdolab/dafoam/blob/main/tests/runTests_DAPimpleFoam.py We are working on a more detailed tutorial for unsteady optimization.

**Reply by erkhans** [reply]
Thank you very much.
